src/test/resources/com/puppycrawl/tools/checkstyle/filters/suppressionpatchxpathfilter/Regexp/newline/defaultContextConfig.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0"?>
+<!DOCTYPE module PUBLIC
+          "-//Puppy Crawl//DTD Check Configuration 1.3//EN"
+          "https://checkstyle.org/dtds/configuration_1_3.dtd">
+<module name="Checker">
+    <module name="TreeWalker">
+        <module name="Regexp">
+            <property name="format" value="// This code is copyrighted\n// \(c\) MyCompany"/>
+        </module>
+
+        <module name="com.puppycrawl.tools.checkstyle.filters.SuppressionPatchXpathFilter">
+            <property name="file" value="${tp}/defaultContext.patch" />
+            <property name="strategy" value="newline" />
+            <property name="neverSuppressedChecks" value="RegexpCheck" />
```

Comment:
Why do you need this property for a newline strategy?

Reply:
Removed, as mentioned in https://github.com/checkstyle/patch-filters/pull/207#issue-460111691, this check is special, its violation line is `0`, so I added it at the beginning. And yes this situation also should be deal with in `context`.